lib/compile/util.js

```diff
@@ -214,20 +214,33 @@ function getPath(currentPath, prop, jsonPointers) {
 }
 
 
+var JSON_POINTER = /^#|\/(?:[^~]|~0|~1)*?$/;
```

Comment:
Cool, I understand. 

Thanks for all your help, by the way. This pull request solved a lot of headaches I was having. If you're ever near San Jose, I'll buy you a beer.